### PR TITLE
Update rw_table type details

### PR DESCRIPTION
**Pull request by ed255**
Small update of the rw_table documentation:
- Update column names to reflect current spec and circuits (using recognizable column names instead of `keyN` names)
- Document the "counter"/"id" types (like `rw_counter`, `txID`, `callID`, etc.) start at 1

### CODE REVIEW

**Comment by ed255** [comment]
@DreamWuGit could help reviewing this small update please?

**Review by ChihChengLiang — APPROVED** [review]
LGTM

**Comment by DreamWuGit** [comment]
> @DreamWuGit could help reviewing this small update please?

@ed255 sure, will have a look !!!

File: specs/tables.md [hunk 1]
```diff
@@ -36,84 +36,87 @@ Proved by the state circuit.
...
-| $counter | false       | TxReceipt                  | $txID    | 0              | PostStateOrStatus          | 0           | $value     | 0          | 0               |
-| $counter | false       | TxReceipt                  | $txID    | 0              | CumulativeGasUsed          | 0           | $value     | 0          | 0               |
-| $counter | false       | TxReceipt                  | $txID    | 0              | LogLength                  | 0           | $value     | 0          | 0               |
+Details:
+
+- **Address (key2)** is reserved for stack, memory, and account addresses.
+- **StorageKey (key4)** is reserved for RLC encoded values
+- **value, valuePrev**: variable size, depending on Tag (key0) and FieldTag (key3) where appropriate.
+- **(rw) counter**: 32 bits, starts at 1.
+- **txID**: 32 bits, starts at 1 (corresponds to `txIndex - 1`).
```
Comment (DreamWuGit) [inline specs/tables.md:45]: `txIndex + 1` ? as I understand `txIndex` starts with zero

Comment (ed255) [reply specs/tables.md:45]: It should be `txIndex + 1`! (I made the same mistake as in logID)

I checked and txID starts at 1. See:
https://github.com/privacy-scaling-explorations/zkevm-specs/blob/8952cf88101e6a6decd6396636e0982eea1931c8/src/zkevm_specs/evm/execution/begin_tx.py#L14-L16

File: specs/tables.md [hunk 2]
```diff
@@ -36,84 +36,87 @@ Proved by the state circuit.
...
+- **address**: 160 bits
+- **callID**: 32 bits, starts at 1 (corresponds to `rw_counter` when the call begins).
+- **Stack -> stackPointer**: 10 bits
+- **Memory -> memoryAddress**: 32 bits
+- **Memory -> value, valuePrev**: 1 byte
+- **storageKey**: field size, RLC encoded (Random Linear Combination).
+- **TxLog -> logID**: 64 bits, starts at 1 (corresponds to `logIndex - 1`), it is unique per tx/receipt.
```
[the remark this answers is not in the document]
Comment (ed255) [reply specs/tables.md]: Thanks! Fixed in https://github.com/privacy-scaling-explorations/zkevm-specs/pull/215/commits/4d4cc7e660d508d3dd1e9db84449519eab926edb

File: specs/tables.md (hunk 1 above)
Comment (ed255) [reply specs/tables.md]: Fixed (the incorrect sign) in https://github.com/privacy-scaling-explorations/zkevm-specs/pull/215/commits/4d4cc7e660d508d3dd1e9db84449519eab926edb

File: specs/tables.md [hunk 3]
```diff
@@ -36,84 +36,87 @@ Proved by the state circuit.
...
-| $counter | true        | TxLog                      | $txID    | $logID         | TopicLength                | 0           | $value     | 0          | 0               |
-| $counter | true        | TxLog                      | $txID    | $logID         | DataLength                 | 0           | $value     | 0          | 0               |
-|          |             |                            |          |                |                            |             |            |            |                 |
-| $counter | false       | TxReceipt                  | $txID    | 0              | PostStateOrStatus          | 0           | $value     | 0          | 0               |
-| $counter | false       | TxReceipt                  | $txID    | 0              | CumulativeGasUsed          | 0           | $value     | 0          | 0               |
-| $counter | false       | TxReceipt                  | $txID    | 0              | LogLength                  | 0           | $value     | 0          | 0               |
+Details:
+
+- **Address (key2)** is reserved for stack, memory, and account addresses.
```
[the remark this answers is not in the document]
Comment (ed255) [reply specs/tables.md:41]: I've updated the PR to show that `log_id` and `index` are packed in address column, and noted that `index` is 32 bits, but let me know what you think! Update in https://github.com/privacy-scaling-explorations/zkevm-specs/pull/215/commits/4d4cc7e660d508d3dd1e9db84449519eab926edb

[the remark this answers is not in the document]
Comment (DreamWuGit) [reply specs/tables.md:41]: Agree,  when I implement tx log state circuit, I realize that 32 bits is enough, I log_id is 32 bits and index is also 32 bits,   so far is good to me, I can update in the future along with tx log state spec change!!

**Review by DreamWuGit — APPROVED** [review]
LGTM, thanks for the update !!

**Pull request merged**